lib/incompatible-packages.js

```diff
@@ -2,9 +2,9 @@
 
 export function checkIncompatible () {
   let incompat = []
-  for (let package of atom.packages.getLoadedPackages()) {
-    if (!package.isCompatible()) {
-      incompat.push(package)
+  for (let package_ of atom.packages.getLoadedPackages()) {
+    if (!package_.isCompatible()) {
+      incompat.push(package_)
```

Comment:
I think this one (suggested by @savetheclocktower) is better, underscores seem quite a bit out of place

```suggestion
  for (let pkg of atom.packages.getLoadedPackages()) {
    if (!pkg.isCompatible()) {
      incompat.push(pkg)
```

Reply:
Thx, this is a great idea.